app/api/audio_services_api.py

```
@@ -68,6 +68,43 @@
+    (``{status, result, metadata, error}``) — in the latter case the inner
+    ``result`` is unwrapped before validation.
+    """
+    try:
+        data = json.loads(payload)
+    except json.JSONDecodeError as e:
+        logger.error("Transcript file is not valid JSON: %s", str(e))
+        raise ValidationError(
+            message=f"Transcript file is not valid JSON: {str(e)}",
+            code="INVALID_TRANSCRIPT_JSON",
+            user_message="The transcript file contains invalid JSON.",
+        )
```

Ruff (0.15.13)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)